LICENSE-binary

```diff
@@ -228,70 +219,41 @@ commons-configuration:commons-configuration:1.6
 commons-digester:commons-digester:1.8
 commons-httpclient:commons-httpclient:3.1
 commons-io:commons-io:2.4
-commons-lang:commons-lang:2.6
-commons-logging:commons-logging:1.2
-commons-net:commons-net:3.1
-com.ning:async-http-client:1.9.21
 com.tdunning:t-digest:3.2
 com.typesafe:config:1.3.2
 com.yammer.metrics:metrics-core:2.2.0
-io.airlift:aircompressor:0.10
 io.dropwizard.metrics:metrics-core:3.2.3
-io.netty:netty:3.9.6.Final
 io.netty:netty-all:4.1.28.Final
 io.swagger:swagger-annotations:1.5.16
 io.swagger:swagger-core:1.5.16
-io.swagger:swagger-jaxrs:1.5.16
-io.swagger:swagger-jersey2-jaxrs:1.5.16
 io.swagger:swagger-models:1.5.16
 io.vavr:vavr:0.9.2
 io.vavr:vavr-match:0.9.2
-it.unimi.dsi:fastutil:6.5.16
-javax.inject:javax.inject:1
+it.unimi.dsi:fastutil:8.2.3
 javax.validation:validation-api:2.0.1.Final
 joda-time:joda-time:2.0
 me.lemire.integercompression:JavaFastPFOR:0.0.13
-net.jpountz.lz4:lz4:1.2.0
 org.apache.avro:avro:1.7.6
 org.apache.commons:commons-compress:1.9
 org.apache.commons:commons-csv:1.0
 org.apache.commons:commons-lang3:3.5
 org.apache.commons:commons-math:2.1
-org.apache.commons:commons-math3:3.2
-org.apache.commons:commons-math3:3.2
-org.apache.curator:curator-client:2.7.1
-org.apache.curator:curator-framework:2.7.1
-org.apache.directory.api:api-asn1-api:1.0.0-M20
-org.apache.directory.api:api-util:1.0.0-M20
-org.apache.directory.server:apacheds-i18n:2.0.0-M15
-org.apache.directory.server:apacheds-kerberos-codec:2.0.0-M15
-org.apache.hadoop:hadoop-annotations:2.2.0
-org.apache.hadoop:hadoop-auth:2.7.0
-org.apache.hadoop:hadoop-common:2.7.0
-org.apache.hadoop:hadoop-hdfs:2.2.0
-org.apache.hadoop:hadoop-mapreduce-client-core:2.2.0
-org.apache.helix:helix-core:0.8.2
```

Comment:
why we are deleting helix? We definitely uses helix..

Reply:
hmm.. something is wrong with the way the automated scripts generate this.... Let me look over again.